av_reallocp_array and av_realloc_array are not the same functions.

Index: libmpcodecs/ve_lavc.c
--- libmpcodecs/ve_lavc.c.orig
+++ libmpcodecs/ve_lavc.c
@@ -442,7 +442,7 @@ static int config(struct vf_instance *vf,
             return 0;
         }
         lavc_venc_context->rc_override=
-            av_reallocp_array(lavc_venc_context->rc_override, i+1, sizeof(*lavc_venc_context->rc_override));
+            av_realloc_array(lavc_venc_context->rc_override, i+1, sizeof(*lavc_venc_context->rc_override));
         lavc_venc_context->rc_override[i].start_frame= start;
         lavc_venc_context->rc_override[i].end_frame  = end;
         if(q>0){
